wacruit/src/apps/resume/repositories.py

```diff
@@ -0,0 +1,65 @@
+from typing import Sequence
+
+from fastapi import Depends
+from sqlalchemy import delete
+from sqlalchemy import select
+from sqlalchemy.orm import exc
+from sqlalchemy.orm import Session
+
+from wacruit.src.apps.resume.exceptions import ResumeNotFound
+from wacruit.src.apps.resume.models import ResumeQuestion
+from wacruit.src.apps.resume.models import ResumeSubmission
+from wacruit.src.database.connection import get_db_session
+from wacruit.src.database.connection import Transaction
+
+
+class ResumeRepository:
+    def __init__(
+        self,
+        session: Session = Depends(get_db_session),
+        transaction: Transaction = Depends(),
+    ):
+        self.session = session
+        self.transaction = transaction
+
+    def get_resumes(self, recruiting_id) -> Sequence[ResumeSubmission]:
+        query = (
+            select(ResumeSubmission)
+            .join(ResumeSubmission.user)
+            .where(ResumeSubmission.recruiting_id == recruiting_id)
```

Comment:
[공식 문서](https://docs.sqlalchemy.org/en/14/orm/loading_relationships.html)도 찾아보고 직접 raw 쿼리 로깅도 해봤는데, 여기서 이렇게 join 해봤자 실제로는 처음 쿼리 실행할 때 의미없는 join이 실행되고, 나중에 user를 가져올 때 다시 한 번 select 문이 수행되는 것 같습니다. 
아래와 같이 바꾸는 게 적절할 듯 싶습니다.
```suggestion
from sqlalchemy.orm import joinedload
...
            select(ResumeSubmission)
            .options(joinedload(ResumeSubmission.user))
            .where(ResumeSubmission.recruiting_id == recruiting_id)
```

Reply:
일단 내용들 전부 반영하기는 하였으나, 아직 자잘한 오류들이 있어  월요일까지는 반영하겠습니다